analysis/01_bfa_iri_seas_forecast.md

```diff
@@ -0,0 +1,645 @@
+# IRI forecast as a trigger for drought in Burkina Faso
+
+This notebook entails the analysis that was done for analyzing the
+IRI seasonal forecast as part of drought-related trigger in Burkina Faso.
+An initial proposal from in-country partners was:
+
+- Trigger #1 in March covering June-July-August. Threshold desired: 40%.
+- Trigger #2 in July covering Aug-Sep-Oct. Threshold desired: 50%.
+- Targeted Admin1s: Boucle de Mounhoun, Centre Nord, Sahel, Nord.
+
+This notebook explores if and when these triggers would be reached.
+As part of this exploration methods for aggregation from raster level
+to the percentage of the area are discussed.
+
+These are the main conclusions:
+
+1) Due to the limited data availability it is almost impossible to set an
+educated threshold. There is only data since 2017.
```

Comment:
The issue is that we don't have historical forecast data. So only forecast data since 2017. We do have rainfall data from before, but since we got the extra parameter of the probability of below average, I cannot come up with a way how we can use historical rainfall date to tune the probability of below average given by the forecast.

Reply:
Oh right true of course :facepalm: